src/main/java/com/java/eval/web/repository/AlbumRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.java.eval.web.repository;
+
+import com.java.eval.web.model.Album;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+
+public interface AlbumRepository extends JpaRepository<Album, Integer> {
+
+    Album findByTitle(@Param("Title")String Title);
```

Comment:
Inutilisé et le @Param ne sert à rien
```suggestion
    Album findByTitle(String Title);
```